src/components/Tasks/TaskForm.vue

```diff
@@ -313,6 +313,10 @@
 			taskId () {
 				return this.projectCategoryId ? null : (this.form?.id || this.modalTaskId || this.$route.params.id)
 			},
+			approximatelyEndTime () {
+				const dt = new Date();
```

Comment:
let's use normal variable names. If you wanted to call it as date, let's use date. It's not a big word